### ISSUE TITLE
Make flexible add more icons

### ISSUE THREAD

**Issue by waja** [Kikobeats/uno-zen#148]
Looking into recent history of Foundation Icons it seems the development has stopped:  https://github.com/zurb/foundation-icons/commits/master

Font Awesome seems to be well maintained and has a much of the recent 'brands' included:
http://fontawesome.io

related:
http://ionicons.com
http://typicons.com
https://mfglabs.github.io/mfglabs-iconset

customize:
https://icomoon.io
http://www.flaticon.com

**Comment by Kikobeats** [comment]
But why? Are you looking for an icon in particular?

**Comment by Kikobeats** [comment]
I don't think that this is a thing that we need (and less urgently).

Currently we get a set of commons icons. When an icon make a rebranding then is moment to think to update, but definitely this works and I'm happy to don't update code that is not broken :-)

**Comment by waja** [comment]
The motivation of this was, that I miss some of the social networks used in the recent days.

**Comment by Kikobeats** [comment]
@waja for example? maybe is my fault because I just use twitter :P

**Comment by waja** [comment]
For example: mixcloud, soundcloud, slack and xing. Just to name a few.

**Comment by Kikobeats** [comment]
:+1: 

I added in your comment more links. I want to make this theme unique, so better have options :)

what do you like more? I like ionicons!

**Comment by waja** [comment]
I've looked into them and mfglabs-iconset looks (beside fontawsome) with most social media icons. Xing (for example) is only available there.
I've suggested fortawesome, cause it looks like it has the most complete support of social icons.

**Comment by Kikobeats** [comment]
@waja ok, go ahead.

Do you think that you can conver this issue into a PR? Steps:

1) Replace foundation reference for fontawesome.
2) Map the same icon to the same css class fonts.
3) Add a set of common icons missing currently.

**Comment by waja** [comment]
As I'm not very familiar with HTML/CSS/JS I can't promise anything or that it will fit you quality standards. I'll have a look into it but ... you know. :-)

**Comment by waja** [comment]
@Kikobeats Looked into the available icons at font-awesome. Do you think it makes sense to ship the foundation-icons AND font-awesome to not break existing customized installations?

**Comment by Kikobeats** [comment]
@waja absolutely. We have to map the new icons to the same css class for backward compatibility and add the missing icons

**Comment by Kikobeats** [comment]
omg, fast click :o

**Comment by waja** [comment]
@Kikobeats Hmm ... to be honest I guess that's way above my skills in web frontend development. At least to share the outcome with others.
